### PR TITLE
RHIDP-4323: OCM permissions require additional permissionsto fully restrict resources

### PR DESCRIPTION
**Pull request by pabel-rh**
**IMPORTANT: Do Not Merge - To be merged by Docs Team Only**

**Version(s):** 1.4

**Issue:** RHIDP-4323

**Reviews:**
- [x] SME: @kim-tsao 
- [x] QE: @PatAKnight @josephca 
- [x] Docs review: @jmagak 
- [ ] Additional review: @hmanwani-rh

### CODE REVIEW

**Review by kim-tsao — APPROVED** [review]

File: modules/authorization/ref-rbac-permission-policies.adoc [hunk 1]
```diff
@@ -181,6 +181,28 @@ Kubernetes permissions::
 
 OCM permissions::
 
+Basic OCM permissions only restrict access to the cluster view but they do not prevent access to the Kubernetes clusters in the resource view.  For more effective permissions, consider applying a conditional policy to restrict access to catalog entities that are of type `kubernetes-cluster`. Access restriction is dependent on the set of permissions granted to a role.  For example, if the role had full permissions (`read`, `update`, and `delete`), then you must specify all its permissions in the `permissionMapping` field.
```
Comment (jmagak) [inline modules/authorization/ref-rbac-permission-policies.adoc:184]:
```suggestion
Basic OCM permissions only restrict access to the cluster view, but they do not prevent access to the Kubernetes clusters in the resource view. For more effective permissions, consider applying a conditional policy to restrict access to catalog entities that are of type `kubernetes-cluster`. Access restriction is dependent on the set of permissions granted to a role. For example, if the role had full permissions (`read`, `update`, and `delete`), then you must specify all its permissions in the `permissionMapping` field.
```

**Review by jmagak — COMMENTED** [review]
LTGM

**Review by Gerry-Forde — APPROVED** [review]
All reviews are completed.

**Pull request merged**